cmd/frontend/backend/own.go

```diff
@@ -0,0 +1,62 @@
+package backend
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/sourcegraph/sourcegraph/internal/api"
+	"github.com/sourcegraph/sourcegraph/internal/authz"
+	"github.com/sourcegraph/sourcegraph/internal/gitserver"
+	"github.com/sourcegraph/sourcegraph/internal/own/codeowners"
+
+	codeownerspb "github.com/sourcegraph/sourcegraph/internal/own/codeowners/proto"
+)
+
+// OwnService gives access to code ownership data.
+// At this point only data from CODEOWNERS file is presented, if available.
+type OwnService interface {
+	// OwnersFile returns a CODEOWNERS file from a given repository at given commit ID.
+	// In the case the file can not be found, `nil` `*codeownerspb.File` and `nil` `error` is returned.
+	OwnersFile(context.Context, api.RepoName, api.CommitID) (*codeownerspb.File, error)
+}
+
+var _ OwnService = ownService{}
+
+func NewOwnService(git gitserver.Client) OwnService {
+	return ownService{git: git}
+}
+
+type ownService struct {
+	git gitserver.Client
```

Comment:
Suggested name change to gitserver. Just `git` makes me think this has to do with the `git` command if I'm reading the usaage of `ownService.git` in the code.

```suggestion
	gitserver gitserver.Client
```

Reply:
Good point about the import clash. 👍🏽

Reply:
👍